api-studdybuddies-backend.js

```diff
@@ -3,7 +3,8 @@ const app = express();
 const cors = require('cors')
 const parametros = require('./src/lib/configParameters');
 
-const userRoute = require('./src/routes/user.route')
+const userRoute = require('./src/routes/user.route');
+const roomRoute = require('./src/routes/room.route');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7172443)
```suggestion
const roomRoute = require("./src/routes/room.route");
```